wagtail_transfer/management/commands/preseed_transfer_table.py

```diff
@@ -0,0 +1,61 @@
+import uuid
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, ImproperlyConfigured
+
+from wagtail_transfer.models import IDMapping, get_model_for_path, get_base_model
+
+
+# Namespace UUID common to all wagtail-transfer installances, used with uuid5 to generate
+# a predictable UUID for any given model-name / PK combination
+NAMESPACE = uuid.UUID('418b5168-5a10-11ea-a84b-7831c1c42e66')
+
+
+class Command(BaseCommand):
+    help = "Pre-seed ID mappings used for content transfer"
+
+    def add_arguments(self, parser):
+        parser.add_argument('model_name', help="Model (as app_label.model_name) to populate table entries for, e.g. wagtailcore.Page")
+        parser.add_argument('--range', help="Range of IDs to create mappings for (e.g. 1-1000)")
+
+    def handle(self, *args, **options):
+        model_name = options['model_name'].lower()
+        try:
+            model = get_model_for_path(model_name)
+        except ObjectDoesNotExist:
+            raise ImproperlyConfigured("%r is not recognised as a model name." % model_name)
+
+        if model != get_base_model(model):
+            raise ImproperlyConfigured(
```

Comment:
I think we should raise a `CommandError` here

Reply:
Makes sense, thanks! Updated in 1025c3ba10788528f11e371500718dc16861cf3a